# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1493182388 14400
#      Wed Apr 26 00:53:08 2017 -0400
# Node ID 054605b58513b09ea0285914c8d29f7e37e7fad6
# Parent  d24d01273bd031b3f805d072177a6098a42a3f2c
don't install options-usage.h

* libinterp/module.mk: Move options-usage.h from the
  octinclude_HEADERS list to the noinst_HEADERS list.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -93,24 +93,24 @@ libinterp_EXTRA_DIST += \
   libinterp/version.in.h \
   $(LIBINTERP_BUILT_DISTFILES)
 
 octinclude_HEADERS += \
   libinterp/builtins.h \
   libinterp/build-env.h \
   libinterp/liboctinterp-build-info.h \
   libinterp/octave.h \
-  libinterp/options-usage.h \
   $(COREFCN_INC) \
   $(LIBINTERP_OPERATORS_INC) \
   $(OCTAVE_VALUE_INC) \
   $(PARSE_TREE_INC) \
   $(PARSER_INC)
 
 noinst_HEADERS += \
+  libinterp/options-usage.h \
   $(NOINSTALL_LIBINTERP_OPERATORS_INC)
 
 nodist_octinclude_HEADERS += \
   libinterp/builtin-defun-decls.h \
   libinterp/corefcn/defaults.h \
   libinterp/corefcn/graphics-props.cc \
   libinterp/corefcn/graphics.h \
   libinterp/corefcn/mxarray.h \
